src/durabletask/azext_durabletask/custom.py

```diff
@@ -1,14 +1,45 @@
 # --------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
-#
-# Code generated by aaz-dev-tools
 # --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
 
-# pylint: disable=too-many-lines
-# pylint: disable=too-many-statements
+import httpx
+from azure.cli.core.commands.client_factory import get_subscription_id
+from .aaz.latest.durabletask.scheduler import Show
 
-from knack.log import get_logger
 
+def list_orchestrations(cmd, resource_group_name, scheduler_name, taskhub_name, max_items=100):
+    # Get FQDN of the scheduler
 
-logger = get_logger(__name__)
+    scheduler = Show(cli_ctx=cmd.cli_ctx)(command_args={
+        "resource_group": resource_group_name,
+        "name": scheduler_name,
+        "subscription": get_subscription_id(cmd.cli_ctx)
+    })
+
+    endpoint = scheduler['properties']['endpoint']
+    endpoint += "/v1/taskhubs/orchestrations/query"
+    grabbed_token = f"Bearer {get_bearer_token(cmd.cli_ctx)}"
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': grabbed_token,
+        'x-taskhub': taskhub_name
+    }
+
+    payload = {"filter": {}, "pagination": {"startIndex": 0, "count": max_items}, "sort": [{"column": "LAST_UPDATED_AT", "direction": "DESCENDING_SORT"}]}
+
+    client = httpx.Client(http2=True)
+    response = client.post(endpoint, json=payload, headers=headers)
+    return response.json()
```

Comment:
Is there an example of what this result looks like on the console?  Does the JSON formatting match the Python and Azure CLI convention or might it need some formatting?

Reply:
For example, the query API returns a set of "trivia" that may or may not make sense in the context of this AZ CLI command, though we'd likely want to return the total count (so that callers know whether it should retrieve the next page).

Reply:
Updated the description to include outputs